src/components/Layout.js

```diff
@@ -5,7 +5,7 @@ import { Link } from 'react-router-dom';
 function Layout() {
   return (
     <div className="navbar">
-      <h1>Bookstore</h1>
+      <h1 className="book-header">Bookstore</h1>
       <nav className="nav-lists">
```

Comment:
- [ ] As I mentioned above :point_up_2:  please use the same colours and font styling as mentioned in [Zaplin](https://zeplin.io/) but still your font styles and colors are totally different from [Zaplin](https://zeplin.io/) design so kindly fix it :+1: check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

| Your Design | Zaplin Design|
|--------|--------|
| ![Screenshot from 2023-05-13 16-04-17](https://github.com/Lul-Abdifan/Bookstore/assets/45226848/b2ffba5f-3d6a-4f7b-9e7f-72902ab9b575) | ![Screenshot from 2023-05-13 16-04-08](https://github.com/Lul-Abdifan/Bookstore/assets/45226848/6c040b7f-a7b4-43da-8d4e-113c54f7d0c4) | 

- [ ] Also if you check your user icon on the right side it's also not the same as [Zaplin](https://zeplin.io/) design :+1:  check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

| Your Design | Zaplin Design|
|--------|--------|
| ![Screenshot from 2023-05-13 16-04-02](https://github.com/Lul-Abdifan/Bookstore/assets/45226848/af0e59bd-66c3-43ee-8331-546ee31f2152)| ![Screenshot from 2023-05-13 16-03-55](https://github.com/Lul-Abdifan/Bookstore/assets/45226848/f98ddf72-000e-42dc-a79b-0c98ea12ab2d) |